book/website/reproducible-research/open/open-source-resources.md

```diff
@@ -0,0 +1,88 @@
+
+The principle of open source resources applies to various typology of digital resources, such as an article, an image, a hardware plan, an educational resource, a video or audio clip. All situations where there is a final format for the user and an editable format manipulated during the production of the resource.
+
+(rr-open-source-whatis)=
+## What Is Open Source?
+
+Commonly associated with software with an available source code, the meaning of this concept is debated and seems to extend beyond software. Defining open source is a delicate question with a variety of interpretations, but it might be summarized as follows:
+
+> Open source is the practice of providing a digital resource with its source files.
+
+At the intersection of these openness movements, we may be on the cusp of a conceptual revolution in the meaning
+of open source. We invite researchers to work on defining and challenging this concept, a knowledge base to explore the idea of an « [Open Source 2.0](https://opensource2.cc/) » is being shaped.
+
+The concept is no longer derived solely from the notion of source code but also from that of source file.
+
+(rr-open-source-for-research)
+## Open Source Resources for an Open Science
+
+(rr-open-source-article)
+### Open Source Article
+
+One of the most basic illustration of open source resources can comes with PDF. This format is not intended to be
+modified and an editable document will be used to generate it. It can be produced from LaTeX files, from an .odt file with
+LibreOffice, from a .docx file with Microsoft Word, from Mardown files using a tool like Quarto and so on.
+
+An open source article will be an article where both formats will be provided, the pdf with its associated source files.
+
+```{figure} ../../../figures/arxiv-article.png
+---
+name: open-source-article
+alt: 'Random ArXiV article to illustrate that the document is provided in pdf format along with downloadable TeX source'
+---
+ArXiV article that provide both a PDF format and its « Tex Source ».
+```
+
+With these source files available, an article can be reworked in collaboration with external parties to produce new versions.
+The modification rights provided by open licenses become enforceable.
+
+(rr-open-source-image)
+### Open Source Image
+
+A similar challenge in managing source files also exists for images, figures and illustrations. An
+image can be made up of multiple layers that can be manipulated individually in an uncompressed format, but really difficult or
+even impossible to modify in a compressed format such as png or jpeg.
+
+Let's take the example of a diagram generated with the mermaid tool in command line mode.
+
+```{figure} ../../../figures/mermaid-diagram.png
+---
+name: open-source-image
+alt: 'Basic diagram used as an example with 3 nodes A, B and C connected together.'
+---
+Diagram in PNG format generated with Mermaid.
+```
```

Comment:
You can add a mermaid diagram directly using the `{mermaid}` directive, rather than adding an image file to the repo. https://mystmd.org/guide/diagrams We have a long-term goal to not store images in the repo to avoid bloat.

Reply:
But not having an image here defeat the purpose of the explanation 😅 Do you have an alternative way to insert images, such as with a CDN ?

It is less problematic to remove the image from ArXiv, for example with a link, rather than this one. The size of the diagram is also quite small (5 Ko)